unpacked_brushes/rf_ui/metadata.xml

```diff
@@ -60,19 +60,16 @@
 	<sprite id="perk_rf_entrenched_mini" ic = "FF23165B" img="skills\mini\perk_rf_entrenched_mini.png" />
 	<sprite id="perk_rf_exploit_opening_mini" ic = "FF23165B" img="skills\mini\perk_rf_exploit_opening_mini.png" />
 	<sprite id="perk_rf_exude_confidence" ic = "FF23165B" img="..\..\gfx\ui\perks\perk_rf_exude_confidence.png" />
-	<sprite id="perk_rf_eyes_up_mini" ic = "FF23165B" img="skills\mini\perk_rf_eyes_up_mini.png" />
```

Comment:
Please also remove the associated mini png file from the `unpacked_brushes\skills\mini` folder.

Reply:
done